resources/migrations/001_update_migrations.yaml

```diff
@@ -3118,6 +3118,15 @@ databaseChangeLog:
                     nullable: true
                     unique: true
 
+  - changeSet:
+      id: v47.00-059
+      author: piranha
+      comment: 'Drops not null from dashboard_tab.entity_id since it breaks drop-entity-ids command'
+      changes:
+        - dropNotNullConstraint:
```

Comment:
will rollback revert the drop not null constraint? some liquibase changes will automatically create a rollback automatically.

if it does I think it's better if we specify a no-op rollback here because there is no benefit to adding it back.

but I guess one downside is you need to make this "drop if exists"